test/listen.test.js

```diff
@@ -16,35 +15,20 @@ test('listen.addMember', t => {
   // NOTE with auto-rebuild active, this listener gets hit twice:
   // 1. first time we see group/add-member (unboxed with DM key)
   // 2. after rebuild
-  listen.addMember(A)(m => {
-    t.equal(m.value.content.root, root, `listened + heard the group/add-member: ${++heardCount}`)
+  function checkRebuildDone (done) {
+    if (A.status().sync.sync) return done()
 
-    if (heardCount <= 2) {
-      t.pass(`addMember called ${heardCount} times`)
-    } else {
-      t.fail(`addMember called ${heardCount} times`)
-    }
+    console.log('waiting for rebuild')
+    setTimeout(() => checkRebuildDone(done), 500)
+  }
```

Comment:
this piece at least is _waaay_ less hacky.

mind you, this whole test might not be needed, as the `rebuild.test.js` implicitly tests this